aiomisc/utils.py

```diff
@@ -46,9 +47,9 @@ def bind_socket(*args, address: str, port: int, options=(),
     sock_addr = address, port
 
     if sock.family == socket.AF_INET6:
-        log.info('Listening tcp://[%s]:%s' % sock_addr)
+        log.info('Listening %s://[%s]:%s', proto_name, *sock_addr)
```

Comment:
If address is IPv6 one, sock_addr will be a 4-tuple.
https://github.com/mosquito/carbon-proxy/pull/5